app/containers/Send/Send.jsx

```diff
@@ -383,8 +383,11 @@ export default class Send extends React.Component<Props, State> {
     }
 
     if (asset !== 'NEO' && asset !== 'GAS') {
-      const decpoint =
-        amountNum.toString().length - 1 - amountNum.toString().indexOf('.')
+      let decpoint = 0
+      if (amountNum.toString().indexOf('.') > -1) {
+        decpoint =
+          amountNum.toString().length - 1 - amountNum.toString().indexOf('.')
```

Comment:
@hal0x2328 can we clean this up to be something like:
```
let decimalPlaces = 0
const amountStr = amountNum.toString();
const decimalPointIndex = amountStr.indexOf('.')

if (decPointIndex !== -1) { // or > -1
  decimalPlaces = amountStr.length - 1 - decPointIndex
}
```

Basically removed the `toString()` x3 and the `.indexOf` x2.

Reply:
Refactored, tested and pushed the suggested changes